src/rtcpxr.c

```diff
@@ -0,0 +1,134 @@
+/**
+ * @file rtcpxr.c  SIP RTCP-XR PUBLISH -- RFC 6035
+ *
+ * Copyright (C) 2010 Creytiv.com
+ */
+#include <re.h>
+#include <baresip.h>
+#include "core.h"
+
+
+static struct sip_lsnr *lsnr;
+static rtcpxr_recv_h *recvh;
+static void *recvarg;
```

Comment:
We are now working to make baresip 100% re-entrant

this adds static/global data and should be avoided. it is better to put them in a struct
that is allocated.